app/javascript/components/purchase-message.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+
+// type Props = {
+//   deliveryCompany: object
+// };
+// defineProps<Props>();
+
+</script>
+
+<template>
+  <div class="my-10 inline-flex h-28 w-full flex-col items-center justify-start gap-4">
+    <div class="text-center text-4xl font-medium leading-10 text-zinc-800">
+      Recibimos tu orden!
+    </div>
+    <div class="self-stretch text-center text-xl font-normal leading-7 text-zinc-800">
+      <!-- // eslint-disable-next-line vue/max-len -->
+      Próximamente podrás ver por acá la fecha de entrega e información de contacto de la empresa que realizará el despacho
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 123. Maximum allowed is 120  vue/max-len